### DISCUSSION TITLE
Highlight current column & row like conoline.vim

### DISCUSSION THREAD

**Discussion by ertect** [andmarti1424/sc-im#676] [Ideas]
The [conoline](https://github.com/miyakogi/conoline.vim) plugin for vim highlights the current line. This would be useful for being able to tell when 2 distant cells are on the same row when reading my spreadsheets with many thin columns & many blank cells yet to be filled.

Just noticed this was already addressed by @andmarti1424 [here](https://github.com/andmarti1424/sc-im/issues/20)

Can void messiness via [row only highlight](https://github.com/andmarti1424/sc-im/issues/20#issuecomment-1005861994)

**Comment by andmarti1424** [comment]
Yeah. Im not sure about adding this as I said earlier.
Convert this to discussion.
